projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -70,11 +62,11 @@ export class PageTimeRangePreferenceService {
       .pipe(shareReplay(1));
   }
 
-  private getDefaultPageTimeRange(path: string): TimeRange {
+  private getDefaultPageTimeRange(path: string): TimeRange | undefined {
     const defaultTimeRange = this.navigationService.getRouteConfig([path], this.navigationService.rootRoute())?.data
```

Comment:
I'm guessing we're assuming the path being called with is a root path given the code here, so we may want to clarify the param name (that this is a specific type of path).